server/index.js

```diff
@@ -29,6 +33,9 @@ app.use('/addPerson', addPerson);
 
 app.use(loggedInOnly); // user must be logged in to access endpoints below
 app.use('/logout', logout);
+app.use('/getAllBooks', getAllBooks);
```

Comment:
Do `app.get` instead of `app.use`. It's better to be specific. Could you go through all of the endpoints and change them to use `app.get` for endpoints that do not change the database (getting data only) and `app.post` for endpoints that do?